MYBackgroundMonitor.m

```diff
@@ -22,6 +22,16 @@ @implementation MYBackgroundMonitor
 @synthesize onBackgroundTaskExpired=_onBackgroundTaskExpired;
 
 
+static BOOL runningInAppExtension() {
+    return [[[[NSBundle mainBundle] bundlePath] pathExtension] isEqualToString: @"appex"];
+}
+
+
+static UIApplication* sharedApplication() {
```

Comment:
This function isn't called anywhere.

Reply:
Sorry. I copied changes from another branch but I didn't copy everything.